app/languages/qml_de.ts

```diff
@@ -820,7 +820,7 @@
         <location filename="../gui/SettingsView.qml" line="1207"/>
         <location filename="../gui/SettingsView.qml" line="1253"/>
         <source>Automatic (Recommended)</source>
-        <translation>Automatisch (empfohlen)</translation>
+        <translation>Automatisch (Empfohlen)</translation>
```

Comment:
this is not a new sentences, it should be small. but i think big is "ok" in this case.